### PR TITLE
Put initial content

### PR DESCRIPTION
**Pull request by rouault**

### CODE REVIEW

**Comment by rouault** [comment]
- Not sure what to put in "Alignment with OGC Standards Baseline". I just refered to the "Relationship to other OGC Standards" paragraph
- Regarding Intellectual Property Rights, for now, I've opted to retain IPR (I'm the unique copyright holder currently of the spec + schema). I'm not opposed to other solutions (like dual licensing MIT - OGC license). But as the PROJ repository is the current host of the spec, it is easier to keep stuff under its MIT license, which shouldn't be an obstacle to adoption/implementation by others.
- I only listed Spatialys in supporting organizations. Others are welcome to join

**Comment by rouault** [comment]
CC @hobu @cholmes @jorisvandenbossche @alasarr

**Comment by hobu** [comment]
* Hobu, Inc. is happy to be listed as a supporting organization.
* pyproj is another implementation (of course it is still using PROJ but IMO most people's exposure to PROJJSON is coming from pyproj). Same for rasterio and Fiona.

**Comment by rouault** [comment]
@snowman2 are you happy with pyproj being mentioned as a PROJJSON implementation and are the information I mentioned OK (see above commit) ?

**Comment by snowman2** [comment]
> @snowman2 are you happy with pyproj being mentioned as a PROJJSON implementation and are the information I mentioned OK (see above commit) ?

Looks good to me :rocket:

**Comment by cholmes** [comment]
Planet is happy to be listed as a supporting organization. cc @qscripter as he's our official OGC rep, so he should give a thumbs up too.

And +1 on Even retaining IPR.

**Comment by qscripter** [comment]
+1 on Planet being a supporting organization and Even retaining IPR.

File: community_std_work_item.adoc [hunk 1]
```diff
@@ -94,19 +143,25 @@ Is the proposed Community standard currently publicly available?
...
 == Supporting OGC Members
 
 ////
 List the supporting organizations. There must be at least three OGC organizations of which at least one must be an OGC Voting Member.
 ////
 
+- Spatialys
+- Hobu, Inc.
+- Planet Labs PBC
+
```
Comment (alasarr) [inline community_std_work_item.adoc]:
```suggestion
- CARTO
```

**Comment by alasarr** [comment]
+1 to add CARTO as a supportive organization

**Comment by rouault** [comment]
merging this pull request as it. Further changes should be issued in new pull requests

**Pull request merged**